loops-and-arrays.js

```diff
@@ -12,7 +12,9 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
+for (let i = 0; i < names.length; i++) {
+    console.log(names[i] + "je");
```

Comment:
Met deze statement log je de koosnaampjes, maar verander je niet de inhoud van de "names" array. Als je buiten de for loop nu dus 'console.log(names)' had gedaan, krijg je de originele waarden van de array. Om de waarden uit de array te veranderen moet je de toewijzing operator: "=" gebruiken: 'names[i] = names[i] + "je"'. of: 'names[i] += "je"'.